data/VT/incentive_relationships.json

```diff
@@ -1,17 +1,25 @@
 {
   "prerequisites": {
-    "VT-29": "VT-27",
-    "VT-30": "VT-28"
+    "VT-2": "VT-1",
+    "VT-4": "VT-3",
+    "VT-8": "VT-7",
+    "VT-10": { "anyOf": ["VT-7", "VT-18"] },
```

Comment:
Green Mountain Power's income bonus for ducted or ductless heat pumps requires the EVT base incentive for either HP type.